### PR TITLE
Richard_Square

### PR DESCRIPTION
**Pull request by RichardSeverich**

### CODE REVIEW

File: src/main/java/org/fundacionjala/coding/richard/Square.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * Given an integral number, determine if it's a square number:
+ * In mathematics, a square number or perfect square is an integer that is the square of an integer; in other words,
+ * it is the product of some integer with itself.
+ */
+public final class Square {
+
+    /**
+     *
+     */
+    public Square() {
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/richard/Square.java]: Remove this unnecessary constructor

File: src/test/java/org/fundacionjala/coding/richard/SquareTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        assertEquals("25 is a square number", true, square.isSquare(25));
+        assertEquals("26 isn't a square number", false, square.isSquare(26));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void shouldWorkForRandomSquareNumbers() {
+        Square square = new Square();
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/coding/richard/SquareTest.java]: You are repeating this variable in all your test methods
please add an @Before method

File: src/main/java/org/fundacionjala/coding/richard/Square.java (hunk 1 above)
Comment (RichardSeverich) [reply src/main/java/org/fundacionjala/coding/richard/Square.java]: Okey I will remove it.

File: src/test/java/org/fundacionjala/coding/richard/SquareTest.java (hunk 2 above)
Comment (RichardSeverich) [reply src/test/java/org/fundacionjala/coding/richard/SquareTest.java]: Okey.

**Comment by carledriss** [comment]
LGTM!

**Pull request merged**